hp/src/ErrorHandler.hs

```diff
@@ -0,0 +1,11 @@
+module ErrorHandler where
+
+import AbstractSyntaxTree (Identifier)
+
+data SemanticError
+    = UndeclaredVariable Identifier
```

Comment:
Maybe you can add a validator so that parameters are entered when calling a function, although it's a bit complicated with the remaining time:

![2024-07-04_21-40](https://github.com/luiggymamani/hufflepuff-own-language/assets/135136456/a71eb007-45ad-4856-ba29-c8ff3e12ec02)